syncclient/client.py

```diff
@@ -43,18 +43,65 @@ def get_browserid_assertion(login, password, fxa_server_url=FXA_SERVER_URL,
     return bid_assertion, hexlify(sha256(keyB.encode('utf-8')).digest()[0:16])
 
 
+class SyncClientError(Exception):
+    """An error occured in SyncClient."""
+
+
+class TokenserverClient(object):
+    """Client for the Firefox Sync Token Server.
+    """
+    def __init__(self, bid_assertion, client_state,
+                 server_url=TOKENSERVER_URL):
+        self.bid_assertion = bid_assertion
+        self.client_state = client_state
+        self.server_url = server_url
+
+    def get_hawk_credentials(self):
```

Comment:
We should add a duration optional parameter to handle this : https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/views.py#L249